### PR TITLE
Add support for Symfony 6

### PR DESCRIPTION
**Pull request by loic425**
Continuation of #119

### CODE REVIEW

File: src/Filesystem/Filesystem.php [hunk 1]
```diff
@@ -15,10 +15,82 @@
 
 use Symfony\Component\Filesystem\Filesystem as BaseFilesystem;
 
-final class Filesystem extends BaseFilesystem implements FilesystemInterface
+final class Filesystem implements FilesystemInterface
```
Comment (lchrusciel) [inline src/Filesystem/Filesystem.php:18]: We've duplicated the contract of this class to run `exists` twice and `getFileContents` once... We should deprecate this class and propose better abstraction (suited for our usecase)

**Review by lchrusciel — APPROVED** [review]

**Comment by lchrusciel** [comment]
Thanks a lot, Jean-Baptiste and Loic! Your rock! 👯

**Comment by alexander-schranz** [comment]
Good work here 👍 

@lchrusciel Do you know if there is a release planned soon? We want to create a release of sulu/theme-bundle with support for Symfony 6 as Sulu with latest [2.5 release](https://github.com/sulu/sulu/releases/tag/2.5.0) is shipped with Symfony 6.

**Comment by loic425** [comment]
> Good work here +1
> 
> @lchrusciel Do you know if there is a release planned soon? We want to create a release of sulu/theme-bundle with support for Symfony 6 as Sulu with latest [2.5 release](https://github.com/sulu/sulu/releases/tag/2.5.0) is shipped with Symfony 6.

@alexander-schranz Just follow our issue for [Symfony 6 support](https://github.com/Sylius/Sylius/issues/13274)

**Comment by alexander-schranz** [comment]
@loic425 I'm aware of that issue but could the syliusthemebundle not released without the sylius core as it does not have any requirements to it.

**Pull request merged**